scripts/mutect2_wdl/mutect2.json

```diff
@@ -17,5 +17,6 @@
   "Mutect2.dbsnp_index": "$__dbsnp_index__",
   "Mutect2.cosmic": "$__cosmic__",
```

Comment:
What happens if there is no PoN?  Are those required?  Will this work?

You should put some general usage information in the top of the M2 WDL and/or in the forum.